app/components/Article/index.js

```diff
@@ -46,6 +50,7 @@ const Article = React.createClass({
         <div ref={(ref) => this.articleRef = ref} id='article-container'>
           {text}
         </div>
+        <button onClick={this.serializeHighlights}>Submit highlights</button>
```

Comment:
This button is a stop gap because fetchNextArticle drives a change in the route which isn't what we want.